### PR TITLE
Implement `looker_user_attribute_group_value` resource

### PR DESCRIPTION
**Pull request by igat64**
In Looker UI it is possible to set the value of user attribute for a group (see the screenshot). Currently, this provider doesn't allow to achieve the same result in Terraform. This PR aims to improve it to support the given use case in our organization.

image

### CODE REVIEW

**Comment by hirosassa** [comment]
@igat64 Thanks for your contribution! I'll take a look this weekend 👍

**Comment by igat64** [comment]
> @igat64 Thanks for your contribution! I'll take a look this weekend 👍

Thanks! Did you have a chance to take a look?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,6 @@ terraform.tfstate.backup
 bin
 vendor
 build
+
+.DS_Store
+.idea
```
Comment (hirosassa) [inline .gitignore:12]: This kind of changes should put in personal git configuration, such as `$HOME/.config/git/ignore`.
Please remove this change.

ref: https://git-scm.com/docs/gitignore

**Review by hirosassa — CHANGES_REQUESTED** [review]
Sorry for late review. I put a small comment, but your changes looks good!

File: .gitignore (hunk 1 above)
Comment (igat64) [reply .gitignore]: Done.

Comment (hirosassa) [reply .gitignore]: Thanks!

**Review by hirosassa — APPROVED** [review]
@igat64 LGTM!

**Comment by igat64** [comment]
> @igat64 LGTM!

Great! When can we expect the new provider version to be deployed to the registry?

**Comment by hirosassa** [comment]
@igat64 I'll release now. Please wait a minutes.

**Comment by hirosassa** [comment]
@igat64 Released! 🎉 
https://registry.terraform.io/providers/hirosassa/looker/0.8.1?pollNotifications=true

**Comment by igat64** [comment]
> @igat64 Released! 🎉 https://registry.terraform.io/providers/hirosassa/looker/0.8.1?pollNotifications=true

Perfect! Thank you.

**Pull request merged**